module7/assignment6.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>First Mobile Example</title>
+  <meta charset="utf-8">
+  <link rel="apple-touch-icon" href="icons/touch-icon2.png">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+  <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+  <link rel="stylesheet" href="_css/themes/theme1.css">
+
+</head>
+<body>
+<!-- Hunter Hackley
+     assignment6.html
+     INFO2439
+     Thoendel
+     10/27/2019
+-->
+
+<div data-role="page" id="third">
+
+    <div data-role="header">
+      <a data-ajax="false" href="test.html" data-icon="home" rel="external">Home</a>
+        <h1><img src="images/home.png" height="50" width="50"><h2>Home</h2></h1>
+    </div>
+
+    <div role="main" class="ui-content">
+
+        <ul id="theList" data-role="listview" data-inset="true">
+
+        </ul>
+
+        <form>
+      <div data-role="fieldcontain">
+        <label for="additemList">Add Item</label>
+        <input type="text" name="additemList" id="additemList" value="" />
+      </div>
+
+        <div data-role="fieldcontain">
+            <label for="additemAddress">New Address</label>
+            <input type="text" name="additemAddress" id="additemAddress" value=""  />
+        </div>
+        <input type="button" id="testBtn" value="Add It">
+        </form>
+    </div>
+
+</div>
+
+<script>
+  $("#testBtn").click(function() {
+    var itemToAdd = $.trim($("#additemList").val());
+    if(itemToAdd == "") return;
+    $("#theList").append("<li>"+itemToAdd+"</li>");
+    $("#theList").listview("refresh");
+});
+$("#testBtn").click(function() {
+    var itemToAdd = $.trim($("#additemAddress").val());
+    if(itemToAdd == "") return;
+    $("#theList").append("<li>"+itemToAdd+"</li>");
+    $("#theList").listview("refresh");
+});
```

Comment:
Looks like you and Ben maybe worked together. I'm going to touch base with him today - would be great if you could sit in too. Ideally - these should be one function - rather than having 2 functions connected to the same event. additemList should be the text of a link and additemAddress should be the href.